test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -596,7 +599,29 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 					"vm.stat_interval":              "10",
 				}
 				kernelParameters := []string{noHzParam, "tsc=nowatchdog", "nosoftlockup", "nmi_watchdog=0", "mce=off", "skew_tick=1"}
-				checkTunedParameters(workerRTNodes, stalldEnabled, sysctlMap, kernelParameters, rtKernel)
+
+				wg := sync.WaitGroup{}
+				By("Waiting for TuneD to start on nodes")
+				for i := 0; i < len(workerRTNodes); i++ {
+					node := &workerRTNodes[i]
+					go func() {
```

Comment:
I did a quick scan of all the new inline go routines you added here and it seems the are repeating themselves.
Can we extract the function (bring back checkTunedParameters in a suitable format)  and use a go routine that calls it instead ?  
If needed you can pass in the function the workgroup with a pointer (wg *sync.WaitGroup) and an interchanging string message

Reply:
We can do something about it sure. let's just see if it helps first.
The test takes ~4 hours so every new push should with a good reason.